internal/api/resources_test.go

```diff
@@ -257,15 +257,15 @@ func TestPutResource(baseT *testing.T) {
 
 		// expect error when CheckResourceAllowed fails
 		m, _ := h.Team.ForAssetType("foo")
-		m.(*plugins.AssetManagerStatic).CheckResourceAllowedFails = true
+		m.(*plugins.AssetManagerStatic).CheckResourceAllowedFails = true //nolint:errcheck
```

Comment:
A short comment would be nice

Reply:
As discussed yesterday, this one (like the vast majority of lints introduced in the new errcheck version) occur in test files. I don't really care to document each one specifically, since it's always the same pattern of casting from an interface to the concrete mock that is instantiated further up in the test.

Reply:
Resolved by https://github.com/sapcc/go-makefile-maker/pull/211